app/src/main/java/org/sopt/androidassignment1/SignInActivity.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.androidassignment1
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidassignment1.databinding.ActivitySigninBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySigninBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySigninBinding.inflate(layoutInflater)
+
+        setContentView(binding.root)
+
+        var id = binding.idInput.text
+        var pw = binding.pwInput.text
+
+        binding.btnLogin.setOnClickListener {
+            if (id.isNotEmpty() && pw.isNotEmpty()) {
+                val intent = Intent(this, HomeActivity::class.java)
+                startActivity(intent)
+            } else if (id.isEmpty() || pw.isEmpty()) {
+                Toast.makeText(this, "로그인 실패", Toast.LENGTH_SHORT).show()
+            }
```

Comment:
```suggestion
            if (id.isNotEmpty() && pw.isNotEmpty()) {
                val intent = Intent(this, HomeActivity::class.java)
                startActivity(intent)
            } else {
                Toast.makeText(this, "로그인 실패", Toast.LENGTH_SHORT).show()
            }
```
else만 써도 괜찮을 것 같아요~!!